### ISSUE TITLE
Low risk message during active infection

### ISSUE THREAD

**Issue by heinp** [corona-warn-app/cwa-wishlist#777]
## Avoid duplicates
* [ x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [ x] Bug is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [x ] Bug is not already reported in another issue

## Technical details

- Device name: Huawei P30 Pro
- Android version: 10
- EMUI Version: 11
- App version: 2.16.2

## Describe the bug

Push notification of lowered risk during active infection. Might be confusing. 

## Steps to reproduce the issue

1. Get red warning
2. Get infected and activate positive test in App
3. Let 14 days pass after step 1

## Expected behaviour

No message of "low risk" during infection. 

## Possible Fix

Check infection status before sending message. 

## Additional context

---
Internal Tracking ID: [EXPOSUREAPP-11970](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11970)

**Comment by Jo-Achim** [comment]
@MikeMcC399 wrote:
> After the infection is known about, the person is expected to isolate. Once the person has recovered and is mixing with other people again, then exposure logging can be reactivated. See FAQ [How do I tell the app that I recovered from COVID-19?](https://www.coronawarn.app/en/faq/#recovered).

_A little note to keep it from getting too easy:_
Quarantine is no longer necessary in all cases, so the previous conclusion that a 'positive' tested person is in quarantine (self-isolation) for a period of time or until a negative test is available is no longer unconditionally valid at the moment.

See: ["Gemäß Bund-Länder-Beschluss vom 7. Januar 2022 sind folgende geimpfte und genesene Personen von dieser Quarantänepflicht für Kontaktpersonen befreit..."  /  "According to the federal-state decision of January 7, 2022, the following vaccinated and recovered people are exempt from this quarantine obligation for contact persons..."](https://www.infektionsschutz.de/coronavirus/fragen-und-antworten/quarantaene-und-isolierung/#tab-5057-4).

**Comment by Ein-Tim** [comment]
@Jo-Achim 

Are you sure? I think if you yourself are tested positive, you have to isolate. 

Only contact persons don't have to go into quarantine under some circumstances.

**Comment by Jo-Achim** [comment]
It is right: actually "contakt persons". But are these contacts less contagious? 
Sorry, but this is a little confusing.

**Comment by MikeMcC399** [comment]
@Jo-Achim 

As @Ein-Tim pointed out, this is about infected people, not simply contact people.

Just go to your link https://www.infektionsschutz.de/coronavirus/fragen-und-antworten/quarantaene-und-isolierung/#tab-5057-4 and read the first entry:

![image](https://user-images.githubusercontent.com/66998419/151795717-2079a2fe-100d-4d38-bd0e-cbe587687647.png)

**Comment by Jo-Achim** [comment]
Sorry and thanks @MikeMcC399, then I'll delete my posts.

**Comment by Ein-Tim** [comment]
@dsarkar I consider this as a bug and not as an enhancement request. I will open a bug report soon.

**Comment by Ein-Tim** [comment]
https://github.com/corona-warn-app/cwa-documentation/issues/856 opened as a bug report.

**Comment by larswmh** [comment]
Hey @heinp, thanks for your report. We have created an internal ticket for it and will raise this topic internally.
Internal Tracking ID: [EXPOSUREAPP-11970](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11970)

---
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
iOS PR addressing this problem: https://github.com/corona-warn-app/cwa-app-ios/pull/4336

**Comment by Ein-Tim** [comment]
Android PR: https://github.com/corona-warn-app/cwa-app-android/pull/4938

**Comment by Ein-Tim** [comment]
This has been fixed in version 2.20 and should be closed now @heinp.
